.linthtmlrc.json

```diff
@@ -1,3 +1,49 @@
 {
-  "extends": "@mate-academy/linthtml-config"
```

Comment:
If you delete the "extends": "@mate-academy/linthtml-config" line, the .linthtmlrc.json file will still work, but you will lose the rules and settings that are defined in the @mate-academy/linthtml-config package. This may affect the quality of your HTML code, as you will no longer have the automatic validations and rules that this configuration provides.